model/config_repo.py

```diff
@@ -23,6 +23,13 @@ class KubernetesSecretTarget:
     secret_key: str
 
 
+@dataclasses.dataclass
+class SugarSolinasServiceTarget:
+    type: str
+    team_id: str
```

Comment:
As `team_id` is used as sugar config name I suggest a more expressive name, e.g. `sugar_cfg_name`.
wdyt?

Reply:
added `sugar_config_name` field